### PR TITLE
feat(control-components): audit dark theme colors

### PR DESCRIPTION
**Pull request by chasestarr**
#### Description

Aligns these components closer to dark theme colors in figma

#### Scope

- [x] Minor: New Feature

### CODE REVIEW

File: documentation-site/static/examples/radio/disabled.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Radio, RadioGroup} from 'baseui/radio';
+
+export default function Disabled() {
+  return (
+    <RadioGroup name="radio group" value="1">
+      <Radio disabled value="1">
```
Comment (nadiia) [inline documentation-site/static/examples/radio/disabled.js]: adding `disabled` prop to the `RadioGroup` should make all the radios disabled

File: src/checkbox/styled-components.js [hunk 2]
```diff
@@ -226,12 +226,20 @@ export const Toggle = styled('div', props => ({
...
+      return props.$theme.colors.sliderTrackFillDisabled;
+    }
+
+    if (props.$isActive && props.$checked) {
+      return props.$theme.colors.sliderTrackFillSelectedActive;
+    }
+
+    if (props.$isHovered && props.$checked) {
+      return props.$theme.colors.sliderTrackFillSelectedHover;
+    }
```
Comment (nadiia) [inline src/checkbox/styled-components.js]: Should there be another condition for just `props.$checked` case (when not active or hovered)?

File: src/slider/styled-components.js [hunk 3]
```diff
@@ -38,18 +38,15 @@ Track.displayName = 'StyledTrack';
...
   return {
     borderRadius: $theme.borders.useRoundedCorners ? borders.radius100 : '0px',
     background: getTrackBackground({
       values: $value,
       colors:
         $value.length === 1
-          ? [$disabled ? colors.mono600 : colors.primary, colors.mono400]
-          : [
-              colors.mono400,
-              $disabled ? colors.mono600 : colors.primary,
```
Comment (nadiia) [inline src/slider/styled-components.js]: Don't we want to keep color change in a disabled state?

File: src/checkbox/styled-components.js (hunk 2 above)
Comment (chasestarr) [reply src/checkbox/styled-components.js]: updated the colors on this element. checked/default have the same color, so no need to create another if statement

File: src/slider/styled-components.js (hunk 3 above)
Comment (chasestarr) [reply src/slider/styled-components.js]: based on figma designs, disabled is indicated by drag handle color

File: documentation-site/static/examples/radio/disabled.js (hunk 1 above)
Comment (chasestarr) [reply documentation-site/static/examples/radio/disabled.js]: thanks!

**Comment by chasestarr** [comment]
merged, but can make any updates in a follow-up pr

**Pull request merged**